src/community/schemaless-features/schemaless-core/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.geoserver.community</groupId>
+        <artifactId>gs-schemaless-features</artifactId>
+        <version>2.20-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.geoserver.community</groupId>
+    <artifactId>gs-schemaless-core</artifactId>
+    <packaging>jar</packaging>
+    <name>Schemaless Features Core</name>
+    <description>Schemaless Features core functionality DB agnostic</description>
+
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-main</artifactId>
+            <version>${gt.version}</version>
+        </dependency>
+
+        <!--   Third-party dependencies   -->
+        <!--   version number defined here as this is a community module -->
+
+        <!--   Test dependencies   -->
```

Comment:
Are `gs-wms` and `gs-wfs` test deps? If so, the scope needs update. If not, perhaps put those above this comment.

Reply:
fixed